docker/run.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
 
 set -e
-
-java $JAVA_OPTS -jar /cromwell/cromwell-*.jar server
+java $JAVA_OPTS -Djava.library.path=./native -javaagent:/cromwell/cromwell-*.jar -jar /cromwell/cromwell-*.jar server
```

Comment:
Also shamelessly stolen from Agora, so it at least doesn't blow up in our standard run environments. I'm fine not doing it that way.